src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,76 @@
+import React, {useEffect, useState} from 'react';
+import {Link, NavLink, useParams} from "react-router-dom";
+import axios from "axios";
+import './Subreddit.css'
+
+function Subreddit() {
+    const {subredditId} = useParams();
+    const [reddit, setReddit] = useState([]);
```

Comment:
Als `reddit` uiteindelijk een object wordt, dan zou ik de default waarde aanpassen naar `undefined`. Het is in ieder geval geen array 😉